src/components/operations/operation-list/react/runtime/OperationListGql.tsx

```diff
@@ -0,0 +1,150 @@
+                }
+            }).finally(() => setWorking(false));
+        }
+    }, [apiName]);
+
+    const getGraphValues = async (): Promise<{graphqlTypes: TGraphqlTypes, availableGraphqlTypes: string[]}> => {
+        let graphqlTypes: TGraphqlTypes;
+        let availableGraphqlTypes: string[];
+
+        try {
+            graphqlTypes = await graphqlService.getGraphqlTypes(apiName);
+            availableGraphqlTypes = await graphqlService.getAvailableGraphqlTypes(graphqlTypes);
+        } catch (error) {
+            throw new Error(`Unable to get GraphQL types. Error: ${error.message}`);
+        }
+
+        return {graphqlTypes, availableGraphqlTypes};
+    }
+
+    const selectOperation = (operation): void => {
+        if (!operation) return;
+
+        allowSelection && setSelectedOperationName(operation.name);
+        const operationUrl = routeHelper.getGraphReferenceUrl(apiName, selectedGraphqlType.toLowerCase(), operation.name, detailsPageUrl);
+        router.navigateTo(operationUrl);
+    }
+
+    const filterOperations = (searchText?: string): void => {
+        if (!searchText) {
+            setFilteredOperations(selectedGraphqlTypeOperations);
+        } else {
+            const filteredOperations: {} = Object
+                .values(selectedGraphqlTypeOperations)
+                .filter(operation => operation["name"].toLowerCase().includes(searchText.toLowerCase()))
+                .reduce((obj: object, item: any) => {
+                    return {
+                        ...obj,
+                        [item.name]: item
+                    };
+                }, {});
+            setFilteredOperations(filteredOperations);
+        }
+    }
+
+    const renderOperation = (operation): JSX.Element => (
+        <Stack
+            key={operation.name}
+            horizontal
+            className={`operation ${operation.name === selectedOperationName && `is-selected-operation`}`}
+            onClick={() => selectOperation(operation)}
+        >
+            {operation.name === selectedOperationName 
+                ? <Body1Strong className={`operation-name${!wrapText ? " nowrap" : ""}`}>
+                        {operation.name}
+                  </Body1Strong>
+                : <Body1 className={`operation-name${!wrapText ? " nowrap" : ""}`}>
+                        {operation.name}
+                  </Body1>
+            }
+        </Stack>
+    )
+
+    return (
+        <div className={"operation-list-container"}>
+            <Stack horizontal verticalAlign="center">
+                <Body1Strong block className={"operation-list-title"}>Operations</Body1Strong>
+                <Button
+                    icon={<ChevronUpRegular />}
+                    appearance={"transparent"}
+                    className={`collapse-operations-button${isCollapsed ? " is-collapsed" : ""}`}
+                    onClick={() => setIsCollapsed(!isCollapsed)} 
+                />
```

Comment:
Maybe this might be one of the buttons which are flagged by the a11y check? I guess adding a title or an aria- tag could resolve it?

Reply:
Thank you. I found and fixed all accessibility issues.